tools/explorer/CMakeLists.txt

```diff
@@ -18,7 +18,9 @@ ExternalProject_Add(
 add_custom_target(explorer
   COMMENT "Building tt-explorer... ${TTMLIR_BIN_DIR}"
   COMMAND pip install $<$<CONFIG:Debug>:-e> ${CMAKE_CURRENT_SOURCE_DIR}/tt_adapter
-  COMMAND pip install ${CMAKE_CURRENT_SOURCE_DIR}/model-explorer/src/model-explorer/src/server/package
+  COMMAND pip uninstall -y ai-edge-model-explorer
+  COMMAND ${CMAKE_COMMAND} -E remove_directory ${CMAKE_CURRENT_SOURCE_DIR}/model-explorer/src/model-explorer/src/server/package/build
+  COMMAND pip install --upgrade ${CMAKE_CURRENT_SOURCE_DIR}/model-explorer/src/model-explorer/src/server/package
```

Comment:
Just for my context, why is this needed?

Reply:
Explorer creates some weird build artifacts and I was getting behaviour where cached versions of the frontend would be propagated from build, so this is to clear and delete any build artifacts to prevent that.